src/components/index.js

```diff
@@ -1,4 +1,5 @@
 // /* eslint-disable import/prefer-default-export */
 import '../assets/index.css';
 export { default as DdButton } from './buttons/index.vue';
+export {default as ImgAvatar } from './avatars/index.vue';
```

Comment:
name should be DdAvatar

Reply:
Export file name changed to "DdAvatar"
<img width="1074" alt="image" src="https://user-images.githubusercontent.com/85391382/217443910-a856b3b6-faf0-4197-b295-f7dd34e911f1.png">